stone-payments/kong-plugin-template-transformer
# Bugfix/Logs Severities

## Description
- Fixes log severities to better represent information.

## How Has This Been Tested?
- `sudo make install && sudo kong restart -vv`
- `tail /usr/local/kong/logs/error.log -f`
- Execute any request to your Kong instance.
- See that the logs does not appear as notice.
